monitoring/mongodb/alerts.yaml

```
@@ -139,3 +139,68 @@ groups:
     annotations:
       description: 'MongoDb has low disk space'
       summary: 'MongoDb has low disk space'
+  
+  - alert: MongoDbPodRecovering
+    expr: |
+      mongodb_rs_members_state{namespace="${namespace}", pod=~"${service}.*"} == 3
+    for: 1h
+    labels:
+      severity: warning
+    annotations:
+      description: "MongoDB pod `{{ $labels.pod }}` has been stuck in the 'RECOVERING' state for the past hour. The instance may not be able to join the replica set if the platform ingests a large number of operations during this time. This alert is expected if the 'Resync a Data Services MongoDB Member' procedure has recently been executed."
```

> The instance may not be able to join the replica set if the platform ingests a large number of operations during this time.

this is alarmist, it can happen but not the most important.
the real issue is that redondancy is reduced.

```suggestion
      description: "MongoDB pod `{{ $labels.pod }}` has been in 'RECOVERING' state for the past hour. This may be expected if the 'Resync a Data Services MongoDB Member' procedure has recently been executed."
```